### DISCUSSION TITLE
config/frontmatter/basic/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#318] [docs-comment]
# config/frontmatter/basic/

概述 通用 Frontmatter 配置，适用于 所有的文章。 示例： 配置 pageLayout 类型： false | 'home' | 'doc' | 'custom' | 'page' | 'friends' | string 默认值： doc 详情： 页面布局方式。 'home': 主页布局 'doc': 文章页布局，包括左侧侧边栏、文章内容...

https://theme-plume.vuejs.press/config/frontmatter/basic/

**Comment by hanxry** [comment]
notes 文件夹下面的 permalink 的生成好像有点小问题。只拼接了NotesOptions 的 link，没有拼接上 NoteItem 中的 link。

**Reply by pengzhanbo** [reply]
都有拼接上的，前提是 notes 要先配置好，主题才能读取解析，以及 如果文档的 permalink 已经存在了，主题是不会去覆盖它们的。

**Reply by hanxry** [reply]
我担心自己配置错了，特意重新用 pnpm create vuepress-theme-plume@latest  构建了演示案例。在什么都不修改的情况下，启动项目，在demo文件夹下面创建新的md文件。期望生成的 permalink 是/demo/xxxx，实际生成的没有demo前缀，只有/xxxx。

**Reply by pengzhanbo** [reply]
我新建项目，但依然未复现出你说的情况。如果方便的话，请通过 [new mini reproduction](https://v2.vuepress.vuejs.org/new) 建立一个最小重现 给我排查问题。

**Comment by edge-sky** [comment]
我想要编写一个工具页，但不想被索引为文章或者笔记，只通过链接访问，类似 友链 的效果，但里面的组件由我自己编排，是否有相关的配置属性？

**Reply by edge-sky** [reply]
使用 draft: true，已经解决了

**Comment by Easton119** [comment]
请问下，关于博客的右侧大纲部分，我在chrome浏览器能正常呈现，但是在 Edge浏览器没正常呈现，是为什么呢

**Reply by pengzhanbo** [reply]
未发现 Edge 异常，请给出你的 Edge 版本，以及异常渲染的截图。

**Reply by Easton119** [reply]
我找到问题了，可能是笔记本屏幕小了，在100%的缩放度下没有显示。如果缩放到90%能显示了😢😢

**Reply by pengzhanbo** [reply]
在屏幕宽度较小时，大纲会隐藏，并在页面上方显示一个页内导航。

**Comment by Jiogei** [comment]
您好，想问下lastUpdated开启后，所有开启的文章都会使用统一的最后一次git提交时间吗？是否可以按文章区分更新时间？我这边每个文章都走了最后一次git提交时间，即时这次提交没有这篇文章都变更～

**Reply by pengzhanbo** [reply]
`lastUpdated` 就是根据文章单独计算的。
